templates/crate/builds.html

```diff
@@ -68,12 +72,15 @@
                                         {%- endif -%}
                                     </div>
                                 </div>
+                            {%- if build.build_status != "in_progress" %}
                             </a>
+                            {%- else -%}
+                            </div>
```

Comment:
Instead of this if/else condition, wouldn't it be better to store what tag we're closing and display it? Much less chances to forgot to update this part of the template if we change this code.